test/Validator/AuthenticationTest.php

```diff
@@ -211,12 +211,7 @@ public function testNoServiceThrowsRuntimeException()
 
     public function testEqualsMessageTemplates()
     {
-        $validator = $this->validator;
-        $this->assertAttributeEquals(
-            $validator->getOption('messageTemplates'),
-            'messageTemplates',
-            $validator
-        );
+        $this->assertGreaterThan(0, count($this->validator->getOption('messageTemplates')));
```

Comment:
I welcome guidance on this change. What does the previous test confirm, and does this change suffice?

Reply:
maybe like this?
```suggestion
        $r = new ReflectionObject($validator);
        $p = $r->getProperty('messageTemplates');
        $p->setAccessible(true);
        $this->assertEquals($validator->getOption('messageTemplates'), $p->getValue($validator));
```